### PR TITLE
fix: user cannot insert the `$$` text due to custom inline syntax

### PR DESCRIPTION
**Pull request by js87zz**
### Please check if the PR fulfills these requirements
- [x] It's the right issue type on the title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has a description of the breaking change

### Description
**as-is**
images

**to-be**
스크린샷 2021-09-06 오후 12 16 24

* related issue(#1768)

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

File: libs/toastmark/src/html/baseConvertors.ts [hunk 1]
```diff
@@ -223,16 +225,32 @@ export const baseConvertors: HTMLConvertorMap = {
...
+    const nomalizedInfo = info.trim().toLowerCase();
+    const customConvertor = convertors![nomalizedInfo];
+    const { entering } = context;
 
     if (customConvertor) {
-      return customConvertor!(node, context);
+      try {
+        return customConvertor!(node, context);
+      } catch (e) {
+        console.warn(e);
```
Comment (adhrinae) [inline libs/toastmark/src/html/baseConvertors.ts:237]: 라이브러리이니만큼 약간은 메세지를 데코레이션 해줘야 하지 않으려나요?
"[tui.editor] 무슨 컨버터 써서 문법 변환하다 오류 났다" 같은 느낌으로요.

**Review by adhrinae — APPROVED** [review]
리뷰 완료합니다.

**Review by jwlee1108 — APPROVED** [review]
리뷰 완료합니다.

**Review by lja1018 — APPROVED** [review]
리뷰완료합니다.

File: libs/toastmark/src/html/baseConvertors.ts (hunk 1 above)
Comment (js87zz) [reply libs/toastmark/src/html/baseConvertors.ts]: 아래처럼 변경했습니다~

`[@toast-ui/editor] - The error occurred when ${info} inline node was parsed in markdown renderer: ${e}`

**Pull request merged**